client/my-sites/themes/themes-site-selector-modal.jsx

```diff
@@ -83,12 +83,12 @@ const ThemesSiteSelectorModal = React.createClass( {
 	 * but only if it also has a header, because the latter indicates it really needs
 	 * a site to be selected and doesn't work otherwise.
 	 */
-	wrapOption( option ) {
+	wrapOption( option, name ) {
 		return Object.assign(
 			{},
 			option,
 			option.header
-				? { action: themeId => this.showSiteSelectorModal( option, themeId ) }
+				? { action: themeId => this.showSiteSelectorModal( name, themeId ) }
```

Comment:
Yep, agree. In a way, that's what I'm fixing with this PR here. If there's an easy solution you can think of off the top of your head, I'd be curious to hear 😄